recipe/meta.yaml

```
@@ -76,11 +82,19 @@ outputs:
         - pip
         - pybind11
         - python
-        - setuptools <=64.0.2 # version requirement from cvxpy
+        - setuptools
       run:
         - ninja
         - numpy
         - python
+      run_constrained:
+        - pennylane-lightning <0.0a0
+      test:
```

Fix indentation level:
```suggestion
    test:
```